extensions/sql-database-projects/src/models/projectEntry.ts

```diff
@@ -44,25 +44,32 @@ export class FileProjectEntry extends ProjectEntry implements IFileProjectEntry
 }
 
 export class DacpacReferenceProjectEntry extends FileProjectEntry implements IDatabaseReferenceProjectEntry {
+	databaseSqlCmdVariableValue?: string;
```

Comment:
I suspect that this class can be transformed into a wrapper for `mssql.DacpacReference`, and many of these properties removed.  Perhaps for a later PR.